mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -20,7 +28,32 @@ AUI.add(
 					initializer: function(config) {
 						var instance = this;
 
-						var todoList = instance.one('.task-container ul');
+						new A.CharCounter(
+							{
+								counter: '.counter',
+								input: '.input-item',
+								maxLength: 35
+							});
+
+						var todoList = instance.byId('todo-list');
+
+						instance._todoList = todoList;
+
+						var finishedTask = instance.byId('finished-task');
+
+						var taskInput = instance.byId('task');
+
+						var taskListLI = todoList.all('li');
```

Comment:
Use camelCase for variable names. Also,```Li``` suffix is clunky and does represent meaning of the content, something like ```taskListItems``` would make more sense.